docs/tutorials/using-qos-ensurance.zh.md

```diff
@@ -174,9 +175,207 @@ spec:
+
+   runningTime：比较两个pod的运行时间，运行时间长的排在后面优先级更高
+
+   如果仅需使用这两个排序策略，使用默认的排序方法即可：会首先比较pod的优先级，之后比较pod对应指标的用量，之后比较pod的运行时长，有一个维度可以比较出结果即为pod的排序结果
+
+   以cpu usage指标的排序为例，还扩展了一些与自身指标相关的排序策略， 如cpu usage 使用量的排序，会依次比较两个pod的优先级，如果优先级相同的情况下，再比较cpu用量，如果cpu用量也相同的情况下继续比较扩展cpu资源用量, 最后比较pod的运行时长，当某一个指标存在差异时即可返回比较结果：`orderedBy(classAndPriority, cpuUsage, extCpuUsage, runningTime).Sort(pods)`
+
+2. 参考水位线和pod用量执行回避动作
+   ```go
+   //将所有触发水位线的metrics根据其Quantified属性区分为两部分
+   metricsQuantified, MetricsNotQuantified := ThrottleDownWaterLine.DivideMetricsByQuantified()
+   // 如果存在不可Quantified的metric，获取具有最高ActionPriority的一个throttleAble的metric对所选择的所有pod进行操作
+   if len(MetricsNotThrottleQuantified) != 0 {
+       highestPrioriyMetric := GetHighestPriorityThrottleAbleMetric()
+       t.throttlePods(ctx, &totalReleased, highestPrioriyMetric)
+   } else {
+       //获取节点和workload的最新用量，构造和水位线差距
+       ThrottoleDownGapToWaterLines = buildGapToWaterLine(ctx.getStateFunc())
+       //如果触发水位线中存在metric的实时用量无法获取，则获取具有最高ActionPriority的一个throttleAble的metric对所选择的所有pod进行压制操作
+       if ThrottoleDownGapToWaterLines.HasUsageMissedMetric() {
+           highestPrioriyMetric := ThrottleDownWaterLine.GetHighestPriorityThrottleAbleMetric()
+           errPodKeys = throttlePods(ctx, &totalReleased, highestPrioriyMetric)
+       } else {
+           var released ReleaseResource
+           //遍历触发水位线的metric中可以量化的metric：如果metric具有排序方法则直接使用其SortFunc对pod进行排序，否则使用GeneralSorter排序；
+           //之后使用其对应的操作方法对pod执行操作，并计算释放出来的对应metric的资源量，直到对应metric到水位线的差距已不存在
+           for _, m := range metricsQuantified {
+               if m.SortAble {
+                   m.SortFunc(ThrottleDownPods)
+               } else {
+                   GeneralSorter(ThrottleDownPods)
+               }
+       
+               for !ThrottoleDownGapToWaterLines.TargetGapsRemoved(m) {
+                   for index, _ := range ThrottleDownPods {
+                       released = m.ThrottleFunc(ctx, index, ThrottleDownPods, &totalReleased)
+                       ThrottoleDownGapToWaterLines[m] -= released[m]
+                   }
+               }
+           }
+       }
+   }
+   ```
+关于扩展自定义指标和排序参考 "自定义指标干扰检测回避和自定义排序" 部分
+
+## 增强的旁路cpuset管理能力
+kubelet支持static的cpu manager策略，当guaranteed pod运行在节点上时，kebelet会为该pod分配指定的专属cpu，其他进程无法占用，这保证了guaranteed pod的cpu独占，但是也造成了cpu和节点的的利用率较低，造成了一定的浪费。
+crane agent为cpuset管理提供了新的策略，允许pod和其他pod共享cpu当其指定了cpu绑核时，可以在利用绑核更少的上下文切换和更高的缓存亲和性的优点的前提下，还能让其他workload部署共用，提升资源利用率。
+
+1. 提供了3种pod cpuset类型：
+- exclusive：绑核后其他container不能再使用该cpu，独占cpu
+- share：绑核后其他container可以使用该cpu
+- none：选择没有被exclusive pod的container占用的cpu，可以使用share类型的绑核
+
+   share类型的绑核策略可以在利用绑核更少的上下文切换和更高的缓存亲和性的优点的前提下，还能让其他workload部署共用，提升资源利用率
+
+2. 放宽了kubelet中绑核的限制
+
+   原先需要所有container的CPU limit与CPU request相等 ，这里只需要任意container的CPU limit大于或等于1且等于CPU request即可为该container设置绑核
+
+
+3. 支持在pod运行过程中修改pod的 cpuset policy，会立即生效 
+
+    pod的cpu manager policy从none转换到share，从exclusive转换到share，均无需重启
```

Comment:
Fix the list style.

<img width="829" alt="image" src="https://user-images.githubusercontent.com/35299017/179461558-058a4cf9-77f1-4132-b8f0-b9c8a93f704b.png">